### DISCUSSION TITLE
Enhancement: inst and voices

### DISCUSSION THREAD

**Discussion by ghost** [FunkinCrew/Funkin#1651] [Ideas]
## What is your suggestion, and why should it be implemented? 
put the opponent's voice in the inst so that when boyfriend misses a note the opponent keeps singing and boyfriend doesn't

**Comment by ghost** [comment]
That's a good idea that I had as well but forgot to post as an enhancement.

**Comment by Windows1Door2Daniel** [comment]
but even if you put into the inst. you only hear bf if you want to heard the voices only

**Comment by ghost** [comment]
> but even if you put into the inst. you only hear bf if you want to heard the voices only

What

**Comment by ghost** [comment]
> @nickpythonluajava said:
> > > but even if you put into the inst. you only hear bf if you want to heard the voices only
> >
> > What

If you want to hear voices only, you won't hear the opponent voice.

**Comment by ghost** [comment]
> > @VMGuy23 said:
> > @nickpythonluajava said:
> 
> > > > but even if you put into the inst. you only hear bf if you want to heard the voices only
> 
> > >
> 
> > > What
> 
> 
> 
> If you want to hear voices only, you won't hear the opponent voice.

well i mean you would hear it in the background and you wouldnt hear it if you turned off the inst in the debug menu
